src/odemis/gui/util/img.py

```diff
@@ -1765,7 +1765,7 @@ def _draw_image_graph(im, size, xrange, xunit, xtitle, yrange, yunit, ytitle, fl
 
     # Draw top/bottom horizontal (wavelength) legend
     text_colour = (0, 0, 0)  # black
-    tick_spacing = size[0] // 4
+    tick_spacing = size[0] // 10
```

Comment:
Since these same spacing devision (the number 10) is used twice I would create a constant.
I think the relation between the tick spacing in x and y direction is more explicit that way.